embeddings/config/flair_config.py

```diff
@@ -0,0 +1,175 @@
+from abc import ABC
+from dataclasses import dataclass, field
+from types import MappingProxyType
+from typing import Any, ClassVar, Dict, Set, Tuple, Union
+
+from embeddings.config.base_config import AdvancedConfig, BasicConfig
+from embeddings.data.io import T_path
+from embeddings.utils.utils import read_yaml
+
+
+@dataclass
+class FlairSequenceLabelingConfigKeys:
+    TASK_MODEL_KEYS: ClassVar[Set[str]] = {
+        "hidden_size",
+        "use_crf",
+        "use_rnn",
+        "rnn_type",
+        "rnn_layers",
+        "dropout",
+        "word_dropout",
+        "locked_dropout",
+        "reproject_embeddings",
+    }
+
+
+@dataclass
+class FlairTextClassificationConfigMapping:
+    TASK_MODEL_KEYS_MAPPING: ClassVar[MappingProxyType[str, Any]] = MappingProxyType(
```

Comment:
Why do we need `MappingProxyType` here? As far as I understand it correctly only first level keys are immutable

Reply:
I described this situation: https://stackoverflow.com/a/58625302

Reply:
Ok, but we do not store nested dictionaries or other objects in these variables so it should do its work.